src/LoveMachine.Core/Config/OscillateConfig.cs

```diff
@@ -0,0 +1,34 @@
+﻿using BepInEx;
+using BepInEx.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace LoveMachine.Core.Config
+{
+    internal static class OscillateConfig
+    {
+        public static ConfigEntry<ConstrictMode> Mode { get; private set; }
```

Comment:
Doesn't look right to reuse constrict mode. I would copy the enum here and rename it to OscillateMode.